tests/behat/features/wip/proposal-support.feature

```diff
@@ -0,0 +1,121 @@
+@wip
+@api
+@javascript
+
+Scenerio: both admin and anoynomous users test the /propsoal support page on ACCCESS Support domain
+
+  Feature: admin user tests page
+    Given I am logged in
+    When I go to /proposal-support
```

Comment:
I don't see a /proposal-support page. Looks like this content is from the home page?